src/struct.h

```diff
@@ -176,7 +182,7 @@ class StructManager {
 
 private:
   std::map<std::string, std::shared_ptr<Struct>> struct_map_;
-  std::unordered_set<std::shared_ptr<Struct>> tuples_;
+  std::unordered_set<std::shared_ptr<Struct>> anonymous_types_;
```

Comment:
naming nit: maybe make this consistent with the method e.g. `anon_types_` or  `AddAnonymousStruct`